tasks.py

```diff
@@ -600,20 +605,8 @@ def e2etest(ctx, name="kind", export=None, kubeconfig=None, system_namespaces="k
     for ns in namespaces:
         run("kubectl -n {} wait --for=condition=Ready --all pods --timeout 300s".format(ns), hide=True)
 
-    try:
-        metallb_config = yaml.load(run("kubectl get configmaps -n metallb-system config -o jsonpath='{.data.config}'", hide=True).stdout)
-    except UnexpectedExit:
-        print("dev-env environment not configured. Try running `inv dev-env -p <layer2/bgp>`")
-        sys.exit(1)
-
-    if metallb_config['address-pools'][0]['name'] == "dev-env-layer2":
-        run("cd `git rev-parse --show-toplevel`/e2etest &&"
-            "go test --provider=local -ginkgo.focus=L2 --kubeconfig={} -service-pod-port={}".format(kubeconfig, service_pod_port))
-    elif metallb_config['address-pools'][0]['name'] == "dev-env-bgp":
-        run("cd `git rev-parse --show-toplevel`/e2etest &&"
-            "go test --provider=local -ginkgo.focus=BGP --kubeconfig={} -service-pod-port={} {}".format(kubeconfig, service_pod_port, opt_skip_docker))
-    else:
-        print("dev-env environment not configured correctly. Try running `inv dev-env -p <layer2/bgp>`")
+    run("cd `git rev-parse --show-toplevel`/e2etest &&"
+            "go test {} --provider=local --kubeconfig={} --service-pod-port={} {}".format(ginkgo_focus, kubeconfig, service_pod_port, opt_skip_docker))    
```

Comment:
I'm a bit on the fence with this change but I guess as we scale out the tests it will make it easier to test the various permutations.

Reply:
What's that makes you not liking it?